policies/0008-contact-information-storage.md

```diff
@@ -0,0 +1,50 @@
+# Where to store contact information
+
+We will store all types of contact information here. It will include
+University, Partners, internal and more contact information.
+
+## Problem Description
+
+We need a private storage location for contact information.
+
+## Policy
+
+The location for this informaiton will be in [ops-private/contacts][1].
+Please store all contact information in this location.
```

Comment:
I propose `Please store all the contact information in individual files using the following format`


## Northeastern Contacts
Mailing List: rchelp@northeastern.edu (needs an NEU account)

### General Contacts:

```
| Person| Role | Email | Phone |
|----|----|----|:----:|
|Name|Title|email|phone|
```
Which will preview as:
| Person| Role | Email | Phone |
|----|----|----|:----:|
|Name|Title|email|phone|

Reply:
alternatively reference a template file?

Reply:
created and referenced template.

Reply:
With the new templates [here](https://github.com/CCI-MOC/ops-private/blob/main/contacts/university_template.md) & [here](https://github.com/CCI-MOC/ops-private/blob/main/contacts/partner_template.md) is this ready to be merged?